src/Libraries/CoreNodeModels/Input/DateTime.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using Dynamo;
+using Dynamo.Models;
+using Dynamo.Nodes;
+using ProtoCore.AST.AssociativeAST;
+
+namespace DSCoreNodesUI
+{
+    [NodeName("Date Time")]
+    [NodeDescription("Create a DateTime object by specifying a date and a time.")]
+    [NodeCategory(BuiltinNodeCategories.CORE_INPUT)]
+    [IsDesignScriptCompatible]
+    public class DateTime : BasicInteractive<System.DateTime>
+    {
+        public DateTime()
```

Comment:
Wait a minute, don't we have this already?  

https://github.com/DynamoDS/Dynamo/blob/6feb670f9bc64c9d6048ac297d0ce30eab1ab01e/src/Libraries/CoreNodes/DateTime.cs